experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -210,12 +210,21 @@ describe('fetch', () => {
     sinon.stub(core.otperformance, 'now').callsFake(() => fakeNow);
 
     function fakeFetch(input: RequestInfo | Request, init: RequestInit = {}) {
+      // Once upon a time, there was a bug (#2411), causing a `Request` object
+      // to be incorrectly passed to `fetch()` as the second argument
+      assert.ok(!(init instanceof Request));
```

Comment:
This is a replace meant for the `init instanceof Request` branch below. We don't have this bug anymore, but this assertion will catch it if it regress, and instantly fail all the tests (synchronously).